src/others/helpers/aid-request-helpers.ts

```diff
@@ -0,0 +1,89 @@
+import { groupBy, map, filter, eq, every } from "lodash";
+import { Location, Supply, AidRequest, ID } from "../contexts/api";
+
+import { ListItem } from "../components/CollapsibleListItem";
+
+export const translateToLocation =
+  (cities: Location[]) =>
+  (city_id: number): Location => {
+    return cities[city_id - 1];
+  };
+
+export const translateToSupply =
+  (supplies: Supply[]) =>
+  (category_id: ID): Supply => {
+    const supply = supplies.find((suply) => String(suply.id) === String(category_id));
+    if (!supply) throw new Error(`Supply category: ${category_id} is not defined`);
+
+    return supply;
+  };
+
+export const sortDates = (a: string, b: string) => {
+  return new Date(a).getTime() - new Date(b).getTime();
+};
+
+export const filterByCategoryIds = (aidRequests: AidRequest[], categoryIds: string[]) => {
+  return filter(aidRequests, (req) => every(categoryIds, (id) => eq(id, req.category_id)));
+};
+
+export const groupByCities = (aidRequests: AidRequest[]) => {
```

Comment:
What do you think about adding return types of those functions just for readability?

Reply:
Done.